pkg/querier/http.go

```diff
@@ -50,7 +50,16 @@ func (q *Querier) RangeQueryHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	query := q.engine.NewRangeQuery(request.Query, request.Start, request.End, request.Step, request.Direction, request.Limit)
+	params := logql.NewLiteralParams(
```

Comment:
Feels a bit weird to have to create a Params to create a Query. Not sure I guess it's not a big deal.